LICENSE

```diff
@@ -1,202 +0,0 @@
-
```

Comment:
Why is the license file being removed?  Many companies requires the presence of a specific license file in order to use OSS software.